src/index.js

```diff
@@ -1,4 +1,7 @@
 import {spawn} from "child_process";
+import {polyfill} from "es6-promise";
+
+polyfill();
```

Comment:
This file can be used as Node.js library also.
So I think that just importing the Promise object is better than writing to global scope.

But `es6-promise.Promise` seems to be the polyfill object even if the native object exists.
Umm...